report/renderer/details-renderer.js

```diff
@@ -398,9 +399,20 @@ export class DetailsRenderer {
     let even = true;
     for (const item of details.items) {
       const rowsFragment = this._renderTableRowsFromItem(item, details.headings);
+
+      // The attribute item.entity could be a string (entity-classification), or
+      // a LinkValue for ThirdPartySummary audit.
+      let entityName = '';
+      if (typeof(item.entity) === 'object' && item.entity.type === 'link') {
+        entityName = item.entity.text;
+      } else if (typeof(item.entity) === 'string') {
+        entityName = item.entity;
+      }
```

Comment:
yeahh typescript makes narrowing here kinda annoying (won't allow the simpler `item.entity?.text ?? item.entity`); so this seems good. Can you remove `entityName` being initialized to empty string tho? Is equivalent since below it would be ignored as falsy, and it better semantically

also: `typeof` is a prefix operator, so you don't need to wrap w/ parentheses